src/Sylius/Bundle/FlowBundle/spec/Process/ProcessSpec.php

```diff
@@ -28,6 +28,11 @@ function it_is_a_process()
         $this->shouldImplement('Sylius\Bundle\FlowBundle\Process\ProcessInterface');
     }
 
+    function it_is_container_aware()
+    {
+        $this->shouldImplement('Symfony\Component\DependencyInjection\ContainerAwareInterface');
```

Comment:
I've just added this to 2.8 upgrade list, only abstract class `ContainerAware` is deprecated, symfony/symfony#17061.